static/js/main.js

```diff
@@ -0,0 +1,85 @@
+$(function() {
+    $(".hide-on-start").hide();
+
+    $(".journal-list-item").on("click", function (event) {
+        event.preventDefault();
+
+        var id = $(this).attr("id").split("-")[1];
+        $("#form-url").val('/edit/' + id);
+
+        $.ajax({
+            method: "GET",
+            url: "/detail/" + id
+        })
+        .done(function(response) {
+            $("#entry-title").html(response.entry.title);
+            $(".twitter-share-button").attr({
+                "data-text": response.entry.title,
+                "data-url": window.location.href + 'detail/' + id
+            });
+            !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');
+            $("#entry-text").html(response.entry.text);
+            $("#journal-list").hide();
+            $(".journal-entry").show();
+        });
+    });
+
+    $("#create-button").on("click", function (event) {
+        event.preventDefault();
+
+        $("#edit-form-header").text("Create Entry");
+        $("#form-url").val('/add');
+        $("#form-title").val('');
+        $("#form-text").val('');
+        $("#journal-list").hide();
+        $(".journal-entry").hide();
+        $("#edit-form-container").show();
+    });
+
+    $("#edit-button").on("click", function (event) {
+        event.preventDefault();
+
+        var url = $("#form-url").val();
```

Comment:
Why not use the `action` attribute of the form element?  You can dynamically update that, read values from it, and so on.  It'd save you carrying this extra hidden form input.  

You know, you can just read the 'action' attribute of the form to get the URL to which the form would have posted:

```html
<form id="editForm" action="/edit/6">
    ...
</form>
```
```javascript
$("form#editForm").on('submit', function(event) {
    var url;
    url = this.attr('action');
    //...
});
```